src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/Shared/CJ4_Shared.js

```diff
@@ -3681,6 +3694,183 @@ class CJ4_PopupMenu_LOWER extends CJ4_PopupMenu_Handler {
+            }
+            this.endSection();
+            this.beginSection();
+            {
+                this.addTitle("L PFD MENU", this.titleSize, 1.0, this.headerColour, true);
+            }
+            this.endSection();
+            this.beginSection();
+            {
+                this.addTitle("FORMAT", this.textSize, 0.4);
+                this.addRadio("ROSE", this.textSize, [CJ4_PopupMenu_Key.PFD_MAP_FORMAT], "L:WT_PFD_MAP_FORMAT");
+                this.addRadio("ARC", this.textSize, [CJ4_PopupMenu_Key.PFD_MAP_FORMAT],"L:WT_PFD_MAP_FORMAT");
+                this.addRadio("PPOS", this.textSize, null);
+            }
+            this.endSection();
+            this.beginSection();
+            {
+                this.addTitle("CONTROLS", this.textSize, 0.5);
+                this.addList("NAV-SRC", this.textSize, ["FMS1", "VOR1", "VOR2"], null);
+                this.addList("RANGE", this.textSize, ["10", "20", "40", "80", "160", "320"], null);
+            }
+            this.endSection();
+            this.beginSection();
+            {
+                this.addSubMenu("BRG SRC", this.textSize, null);
+                this.addSubMenu("CONFIG", this.textSize, null);
+                this.addSubMenu("OVERLAYS", this.textSize, null);
+                this.addSubMenu("RADAR", this.textSize, null);
+                this.addSubMenu("REFS", this.textSize, null);
+                this.addSubMenu("TAWS", this.textSize, null);
+                this.addSubMenu("BARO SET", this.textSize, null);
+            }
+            this.endSection();
+        }
+        this.closeMenu();
+        this.escapeCbk = this.showMainPage.bind(this, 6);
+        this.highlight(_highlight);
+        page.appendChild(sectionRoot);
+        Utils.RemoveAllChildren(this.root);
+        this.root.appendChild(page);
+    }
+    showNavPage() {
+        this._isOnMainPage = false;
+        let page = document.createElementNS(Avionics.SVG.NS, "svg");
+        page.setAttribute("id", "ViewBox");
+        page.setAttribute("viewBox", "0 0 500 500");
+        let sectionRoot = this.openMenu();
+        {
+            this.beginSection();
+            {
+                this.addTitle("PFD MENU", this.titleSize, 1.0, this.previousHeaderColour);
+            }
+            this.endSection();
+            this.beginSection();
+            {
+                this.addTitle("BRG SRC", this.titleSize, 1.0, this.headerColour, true);
+            }
+            this.endSection();
+            this.beginSection();
+            {
+                this.addTitle("BRG PTR 1", this.textSize, 0.5);
+                this.addRadio("OFF", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR1_SRC]);
+                this.addRadio("FMS1", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR1_SRC]);
+                this.addRadioRange("VOR1", this.textSize, 108, 117.95, 0.005, [CJ4_PopupMenu_Key.BRG_PTR1_SRC, CJ4_PopupMenu_Key.BRG_VOR1_FREQ]);
```

Comment:
This code does not appear to be synchronized with the bearing pointer changes.

Reply:
I rebased this branch off 0.6.5. Are bearing pointers on the pre-release yet?

Reply:
They were already on main, but this code looks moved entirely from the original location, so it may not be picking up the conflict.

These lines now should be like this:

```javascript
            this.beginSection();
            {
                this.addTitle("BRG PTR 1", this.textSize, 0.5);
                this.addRadio("OFF", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR1_SRC]);
                this.addRadio("FMS1", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR1_SRC]);
                this.addRadio("VOR1", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR1_SRC]);
                this.addRadio("ADF1", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR1_SRC]);
            }
            this.endSection();
            this.beginSection();
            {
                this.addTitle("BRG PTR 2", this.textSize, 0.5);
                this.addRadio("OFF", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR2_SRC]);
                this.addRadio("VOR2", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR2_SRC]);
                this.addRadio("ADF2", this.textSize, [CJ4_PopupMenu_Key.BRG_PTR2_SRC]);
            }
            this.endSection();
```
Check the original location this stuff was copied from to make sure you picked everything you need up.